### DISCUSSION TITLE
missing char() as DB type

### DISCUSSION THREAD

**Discussion by roaster-vicc** [supabase/supabase#1473] [Questions] [answered]
I'm creating a table, and this is no char type for example char(20).

Also, can I connect using pgadmin?

**Comment by ghost** [comment, marked as answer]
To use `char` as a data type you will need to type manual sql queries. 

```sql
CREATE TABLE new_table ( name CHAR(10) ); 
```

As for connecting using pgAdmin, I think this could be possible as you are provided with the database connection information at _Settings > Database_.

**Reply by awalias** [reply]
yes pgadmin is possible, either by port 5432 or 6543 for pgbouncer

**Comment by steve-chavez** [comment]
Note that It's not recommended to use a `char(n)` type by the official postgres wiki:
https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_char.28n.29.

If you want to validate a fixed length string, a `char(n)` will not enforce that a string length is less than `n`.

You sould always prefer a `text` type plus a CHECK constraint(`CHECK(length(string)=3)`) for doing this.
